electron/debugging/docker.md

```diff
@@ -0,0 +1,52 @@
+## Debugging CI With Docker
+
+Sometimes, you may experience test discrepancies between your local machine and the ones used to run CI, and so want to replicate those failures in an environment that most closely resembles them.
+
+On Linux, that's going to require you to run tests in a Docker container.
+
+1. Ensure you have Docker installed on your Linux machine.
+    * `$ sudo apt install docker.io`
+2. Ensure the Docker daemon is running 
+    * `$ sudo systemctl start docker`
+3. Find Electron's Docker image in our CircleCI configuration - it should look something like:
+    * `electronjs/build:3bba0fdee0d5650e751c2561f1806f9138dfe56a`
```

Comment:
This should have more steps -- where would an end user look for this, and how should they find it?

Also, might be worth spelling out that it's not always going to literally be that hash